qovery/data_source_helm.go

```diff
@@ -470,6 +470,34 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
 					},
 				},
 			},
+			"custom_domains": schema.SetNestedAttribute{
+				Description: "List of custom domains linked to this container.",
+				Optional:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"id": schema.StringAttribute{
+							Description: "Id of the custom domain.",
+							Computed:    true,
+						},
+						"domain": schema.StringAttribute{
+							Description: "Your custom domain.",
+							Computed:    true,
+						},
+						"validation_domain": schema.StringAttribute{
+							Description: "URL provided by Qovery. You must create a CNAME on your DNS provider using that URL.",
+							Computed:    true,
+						},
+						"generate_certificate": schema.BoolAttribute{
+							Description: "Qovery will generate and manage the certificate for this domain.",
```

Comment:
_suggestion_:  `Qovery will generate and manage the certificate for this domain.` => `Qovery will generate and manage domain's certificate.`

Reply:
I will fix your comment in a separate PR to also change the descriptions of the other services (app, container...)